### PR TITLE
finalized the project

### PR DESCRIPTION
**Pull request by Ashe546**
Finalized the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Ashe</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
```
Comment (Maclenn77) [inline index.html]: I can't comment in this merge some of the issues, so I add my comments here.

- [ ] You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

  - Colors.
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.

Consider using the typography in the design guideline, [Cocogoose](https://www.cdnfonts.com/cocogoose.font) and [Lato](https://fonts.google.com/specimen/Lato). Remember that following design guidelines is an important part of the work as front-end developer.

- [ ] You should build only these 2 pages:

  -  [x] The home page.
  - [ ] The about page.

There's not about page in the project. Consider adding the about page and create a link in the navigation bar, as this is part of the requirements.

- [ ] Links
  - [ ] The home page should have a link in the menu to the about page.
  - [ ] The logo in the header links to the home page.

- [ ] Mobile menu
  
There is not mobile menu in the mobile version, and it doesn't appears a hamburguer icon or something similar. It's important to keep functionalities in the mobile version, as it's the most used devise for most people to navigate. Consider fixing this problem. **Hint**: Bootstrap makes really easy to build a [responsive menu bar](https://getbootstrap.com/docs/4.3/components/navbar/).

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+
+## Built With
+
+- Major languages - HTML, CSS and JavaScript
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://ashe546.github.io/capstone-1/)
```
Comment (Maclenn77) [inline README.md]: Nice demo! :+1:  However, it's still missing a video that shows your project. 

- [ ] A video with a maximum length of 5 minutes. The content of the video should include:

 - A description of the project.
 - A demo of the project features (different pages and different screen sizes).
 - You should also highlight some interesting piece of code or something you built that you are very proud of.

This requirement is a big opportunity to practice your communication skills. Don't be shy!

**Review by Maclenn77 — CHANGES_REQUESTED** [review]
Hi @Ashe546 , 

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- I liked the interactive elements in the main program

## Required Changes ♻️
_Check the comments under the review._

Several requirements are missing in the project. Check the comments for more details. Consider reading againt the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md). Most of them are easy to handle. I resume some of them.

- [ ] Procure to follow the original desing (included typography)
- [ ] About page is missing
- [ ] Recording a video is mandatory
- [ ] Navbar disappears in the mobile version

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ashe546** [comment]
loom video [link https://www.loom.com/share/581838d2cef14b28aa8ad75af62d1731](https://www.loom.com/share/1cf415878bb74661b624230efa4ca05e)

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @Ashe546 👋🏻, 

Your capstone looks perfect 😍 after making all the requested changes. Good job 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

you-did-it-good-job


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**